### DISCUSSION TITLE
Weathermap Version 1.0 is not working after upgrade from 0.98a

### DISCUSSION THREAD

**Discussion by gty71** [thurban/plugin_weathermap#112] [Q&A]
Hi,

i just tried to upgrade from v0.98a to v1.0 but fro any reason, after moving all config files to the new weathermpa plugin folder the plugin isn't working.
After starting and openiong the Weathermap tab the plugin throws error 

ERROR PHP ERROR in Plugin 'weathermap': Uncaught Error: Class 'IntlDateFormatter' not found in /usr/share/cacti/site/plugins/weathermap/lib/WeatherMap.class.php:1889 Stack trace: #0 /usr/share/cacti/site/plugins/weathermap/lib/WeatherMap.class.php(2007): WeatherMap->{closure}() #1 [internal function]: WeatherMap->{closure}() #2 /usr/share/cacti/site/plugins/weathermap/lib/WeatherMap.class.php(2009): preg_replace_callback() #3 /usr/share/cacti/site/plugins/weathermap/lib/WeatherMap.class.php(2017): WeatherMap->strftime() #4 /usr/share/cacti/site/plugins/weathermap/lib/WeatherMap.class.php(3609): WeatherMap->DrawTimestamp() #5 /usr/share/cacti/site/plugins/weathermap/lib/poller-common.php(442): WeatherMap->DrawMap() #6 /usr/share/cacti/site/plugins/weathermap/setup.php(1138): weathermap_run_maps() #7 /usr/share/cacti/site/lib/plugins.php(177): weathermap_poller_bottom() #8 /usr/share/cacti/site/lib/plugins.php(74): api_plugin_run_plugin_hook() #9 /usr/share/cacti/site/poller.php(801): api_plugin_hook() #10 {main} thrown in file: /usr/share/cacti/site/plugins/weathermap/lib/WeatherMap.class.php on line: 1889

and plugin is disabled immediately.

Does anyone have the same problem?

cheers
Markus

**Comment by TheWitness** [comment]
Install php-intl then restart httpd.

**Reply by gty71** [reply]
Hi,

thanks for quick reply - I have overseen this.
Doing so things improved, but there are still some errors.

Existing config is working and weathermap is updated - but I'm not able to edit the conf files - always throwing error listed below:

27.03.2023 13:56:58 - CMDPHP ERRORS DETECTED - DISABLING PLUGIN 'weathermap'
--
27.03.2023 13:56:58 - CMDPHP PHP ERROR Backtrace: (CactiShutdownHandler())
27.03.2023 13:56:58 - ERROR PHP ERROR in Plugin 'weathermap': Uncaught Error: Call to undefined function cacti_cookie_set() in /usr/share/cacti/site/plugins/weathermap/weathermap-cacti-plugin-editor.php:305 Stack trace: #0 {main} thrown in file: /usr/share/cacti/site/plugins/weathermap/weathermap-cacti-plugin-editor.php on line: 305



btw - forgot to mention my Cacti version -> 1.2.10

**Comment by TheWitness** [comment]
Minimum Cacti version is 1.2.24.  You could likely get 1.2.23 to work, but not much before that.

**Comment by gty71** [comment]
Ahh - ok - as in the Readme-file it is written 1.2.x I thought also 1.2.10 would work.

Anyway - thanks for fast clarification!

**Comment by TheWitness** [comment]
No problem.  1.2.24 has been a real good upgrade thus far.
